tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -2209,14 +2209,14 @@ def _test_forward_slice_operation_input(input_value, begin_value, size_value):
     with tf.Graph().as_default():
         input_tensor = tf.placeholder(
             shape=input_data.shape, dtype=input_data.dtype, name="input")
-        begin_tensor = tf.expand_dims(begin_value, axis=0)
-        size_tensor = tf.expand_dims(size_value, axis=0)
-        slice_tensor = tf.slice(input_tensor, begin_tensor, size_tensor, name='slice_output')
+        tf.slice(input_tensor, begin_value, size_value, name='slice_output')
         compare_tf_with_tvm([input_data], ['input:0'], 'slice_output:0')
 
 
 def test_forward_slice():
-    _test_forward_slice_operation_input([1, 1], 0, 2)
+    _test_forward_slice_operation_input([1, 1], [0], [2])
```

Comment:
Is it better to add below lines inside `_test_forward_slice_operation_input` to also accept begin_value, size_value input as a single int?
```
    begin_value = [begin_value] if isinstance(begin_value, int) else begin_value
    size_value = [size_value] if isinstance(size_value, int) else size_value
```

Reply:
Well, in the TF documentation is said that tf.slice() arguments **begin** & **size**  both should be tensors. I am aware that the scalars are 0D tensors but if the TF function, in general, doesn't accept scalars, then form my point of view, it doesn't make sense to test something that is not supposed to work like this.